roles/kafka_broker/tasks/main.yml

```
@@ -361,6 +361,30 @@
   tags:
     - log
 
+- name: Validate Kafka Broker Jolokia Access Control Configuration
```

this should be called from top of the file so if there is any issue it fails early